evmos/assetlist.json

```diff
@@ -100,6 +100,33 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/neok.svg"
         }
       ]
+    },
+    {
+      "description": "The token of Teledisko DAO.",
+      "denom_units": [
+        {
+          "denom": "erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
+          "exponent": 0
+        },
+        {
+          "denom": "berlin",
+          "exponent": 18
+        }
+      ],
+      "base": "erc20/0x1cFc8f1FE8D5668BAFF2724547EcDbd6f013a280",
+      "name": "Teledisko DAO",
+      "display": "berlin",
+      "symbol": "BERLIN",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/evmos/images/berlin.svg"
```

Comment:
Please remove the extra padding from the svg image. (should be square dimensions)